src/package.json

```diff
@@ -11,16 +11,16 @@
   },
   "license": "Apache-2.0",
   "devDependencies": {
-    "electron-connect": "^0.3.3"
+    "electron-connect": "^0.4.4"
   },
   "dependencies": {
-    "auto-launch": "^2.0.1",
-    "bootstrap": "^3.3.6",
+    "auto-launch": "^2.1.0",
+    "bootstrap": "^3.3.7",
     "electron-squirrel-startup": "^1.0.0",
     "os-locale": "^1.4.0",
-    "react": "^15.0.1",
-    "react-bootstrap": "~0.29.0",
-    "react-dom": "^15.0.1",
-    "yargs": "^3.31.0"
+    "react": "^15.3.0",
+    "react-bootstrap": "~0.30.2",
```

Comment:
Did you check the updated app? I could not see anything in the settings page.

In my understanding, `react-bootstrap` is a very unstable library. In 0.28 to 0.29, it was definitely changed when I had written the settings page. So I think there is something wrong by using `react-bootstrap@0.30.x`.